manager/controllers/advisory_systems.go

```diff
@@ -61,7 +61,7 @@ func AdvisorySystemsListHandler(c *gin.Context) {
 
 	query, err = ApplySort(c, query, SystemsSortFields...)
 	if err != nil {
-		LogAndRespError(c, err, err.Error())
+		LogAndRespBadRequest(c, err, "sort application failed")
```

Comment:
Could we add a test case for this ? Returning invalid request on invalid sort field.

Reply:
Yes, I would create ticket to increase test coverage in the next sprint. Or do you really want to add it into this PR?

Reply:
Why not, it's a short PR

Reply:
ok, test added